bdio2/src/main/resources/com/blackducksoftware/bdio2/bdio-context-2.1.jsonld

```diff
@@ -154,6 +154,9 @@
       "@id" : "https://blackducksoftware.github.io/bdio#hasPlatform",
       "@type" : "https://blackducksoftware.github.io/bdio#Products"
     },
+    "project" : "https://blackducksoftware.github.io/bdio#hasProject",
```

Comment:
Will this change also update the related specification document accordingly?

Currently here: 
https://blackducksoftware.github.io/bdio/specification/2.1